client/Pages/EventTypeCreate/Update.elm

```diff
@@ -265,6 +265,7 @@ validate model eventTypeStore =
                         |> checkNameFormat model
                         |> isNotEmpty FieldName model
                         |> isNotEmpty FieldSql model
+                        |> isNotEmpty FieldEnvelope model
```

Comment:
You check if it is set as mandatory, but here you say it is optional.
 https://github.com/zalando-nakadi/nakadi-ui/pull/77/files#diff-5efab63378760e1e0c61310e95fb2988R77